pkgs/top-level/all-packages.nix

```diff
@@ -6213,6 +6213,7 @@ with pkgs;
   };
 
   clang = llvmPackages.clang;
+  clang-manpages = llvmPackages.clang-manpages;
```

Comment:
I definitely dislike this because with basically every other package you can just do ```${lib.getOutput "man" pkg}/share/man``` to get the manpages. I guess it's okay as a short term fix but it just seems like something we should be able to fix. What if we just made this ```clang = llvmPackages.clang-manpages```? I think most users whan clang with manpages & this will only be used by people doing ```nix-env -iA nixpkgs.clang``` (stdenv uses llvmPackages directly IIRC).

Reply:
clang and clang-manpages are 2 separate packages.

Reply:
Also nix-env -i clang is considered bad practice and doesn't even work properly.

Reply:
FWIW If we bind `clang-manpages` and `clang-nomanpages`, we can in a "late bind" fashion do the man output trick so that the bootstrap-persisted clang and fresh manpages are combined together

Reply:
Ok I guess it is okay in this instance just because of the complications of bootstrapping/stdenv. But in general asking people to remember things like ```x-manpages``` seems like a bad idea. #24717 is definitely a bug that affects this but we shouldn't let it dictate how these things work.

Reply:
I'm a bit reluctant to do that given the problems it's caused so far. But yes, that's an option.